### PR TITLE
[roadmap epic]: move upgrades pages

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

deletes:
`upgrades/index.md`
`upgrades/get_involved`

moves:
`upgrades/beacon-chain` --> `roadmap/beacon-chain`
`upgrades/merge` --> `roadmap/merge`
`upgrades/merge/issuance` --> `roadmap/merge/issuance`

adds appropriate redirects to `redirects.json`

## Related Issue

#9242 
Fixes https://github.com/ethereum/ethereum-org-website/issues/9762

### CODE REVIEW

**Review by pettinarip — CHANGES_REQUESTED** [review]
One thing I notice here is that we should also move the md files for the other languages as well.

For example, in the Spanish version we will have 2 pages for the merge:
- [/es/upgrades/merge/](https://ethereumorgwebsitedev01-moveupgradespages.gatsbyjs.io/es/upgrades/merge/) - should be deleted
- [/es/roadmap/merge/](https://ethereumorgwebsitedev01-moveupgradespages.gatsbyjs.io/es/roadmap/merge/) - this one is the one we want but is not getting translated because the other one exists

I can help you making these changes @jmcook1186 (not only for Spanish but for all the other langs that have a md file for these pages).

Will let you know when its ready.

File: redirects.json [hunk 1]
```diff
@@ -51,6 +51,34 @@
     "fromPath": "/eth2/",
     "toPath": "/en/roadmap/"
   },
+  {
+    "fromPath": /upgrades/sharding/",
```
Comment (jmcook1186) [inline redirects.json:55]:
```suggestion
    "fromPath": /upgrades/sharding",
```

**Comment by jmcook1186** [comment]
yea the sharding page should be deleted from all languages, and redirected to `roadmap/danksharding`.
I'll add the redirect now.

**Review by pettinarip — APPROVED** [review]
Ok, I've modified the redirects a bit.

Now, per each page we are moving, I've added:
- A redirect from the root path to the English path - e.g. `/upgrades/merge` => `/en/roadmap/merge`
- A redirect to catch when the user specify the language in the path (Netlify only) - e.g. `/[lang]/upgrades/merge` => `/[lang]/roadmap/merge`

Then, I've also updated all the rest of md files (non English) to represent the same naming and structure we have for English.

**Review by corwintines — APPROVED** [review]

**Pull request merged**